docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -380,7 +380,7 @@ Alias: `dateTrunc`.
 
 **Parameters**
 
--   `unit` — The type of interval to truncate the result. [String](../../sql-reference/data-types/string.md).
+-   `unit` — The type of interval to truncate the result. [String Literal](../syntax.md#syntax-string-literal).
```

Comment:
Minor unrelated change which I overlooked this one in my last PR. This is actually string literal as the `unit` is specified as `'hour'` , `'minute'`, `'second'` etc.

Example:
```sql
ELECT now(), date_trunc('hour', now(), 'Europe/Moscow')
```